core/clingutils/src/TClingUtils.cxx

```diff
@@ -4184,6 +4184,8 @@ void ROOT::TMetaUtils::GetNormalizedName(std::string &norm_name, const clang::Qu
    cling::Interpreter::PushTransactionRAII clingRAII(const_cast<cling::Interpreter*>(&interpreter));
    normalizedType.getAsStringInternal(normalizedNameStep1,policy);
 
+   TClassEdit::AtomicTypeNameHandlerRAII atomicTypeNameHandler(normalizedNameStep1);
```

Comment:
This may or may not be doing what is intended.  `normalizedNameStep1` is used only as input to the next step.  The output is `norm_name` which after this addition will **not** contain the `_Atomic(...)` part.

Reply:
Addressed.